CRM/Event/Form/Registration/Confirm.php

```diff
@@ -1325,4 +1348,51 @@ private function calculateAmounts(): array {
     return [$taxAmount, $participantDetails, $individual, $amountArray];
   }
 
+  /**
+   * Interim refactoring extraction.
+   *
+   * @internal
+   * @return array
+   */
+  private function calculateLegacyAmountArray(): array {
+    CRM_Core_Error::deprecatedWarning('Do not access _amount as it will be removed');
```

Comment:
@eileenmcnaughton I'm not sure we need the `deprecatedWarning` here, as one will already be called from the `__get()`, and `calculateLegacyAmountArray` is private so cannot be called directly.